### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by Dresch360** [sdetweil/MMM-SleepWake#8]
Thanks again for all of your help with your module! It's very responsive and interactive. Is there any way to show another module when sleep in sleep mode? For example, when it sleeps and goes to 'hide' mode, instead of a blank screen I would like to show MMM-Unsplash images.

Any help would be greatly appreciated.

Thanks again!

**Comment by PraiseUadiale** [comment]
@Dresch360 Do I need Motion for any of this to work? The usage section is not very clear, can you explain please?

**Comment by sdetweil** [comment]
you need something to signal motion.  my module does not know on its own.

**Comment by PraiseUadiale** [comment]
Thanks for the quick response! I will try that

**Comment by sdetweil** [comment]
@Dresch360 sorry, I missed your other question. no as currently written, it's all or nothing

**Comment by PraiseUadiale** [comment]
@sdetweil Sorry to disturb so to confirm run this "on_event_start /home/pi/MagicMirror/modules/MMM-SleepWake/external_motion started" on the terminal? and I don't need any other software apart from cloning this repo in the modules dir?

**Comment by sdetweil** [comment]
no. that line goes in the motion.conf file for it to trigger the module

I needed to hide/show all the modules, but didn't have a pir sensor. only a webcam.

so my module does the show/hide.  but needs something else to detect motion and inform the module.

the script does the work of informing,. just need something to call it

I didnt want to learn how to do motion detection with different cameras or support that going forward.

**Comment by PraiseUadiale** [comment]
Thank you! I was able to figure it out. Sorry for the dumb question so we do somehting like sudo motion to start up motion and then run MM? @sdetweil

**Comment by sdetweil** [comment]
you could use pm2 to start motion too, but yes, you can also add the sudo run motion to the mm.sh used by pm2.

I do that...check if motion is running, start if not
